File: URI Online Judg/Beginner/URI-C-Plus-Plus/1013.cpp
#include <iostream>

int main()
{
	int a, b, c, maior, result;
	std::cin >> a >> b >> c;

	if (a < b && a < c) {
		if (b < c) {
			result = c;
		}
		else{
			result = b;
		}
	}
	else if (b < a && b < c) {
		if (a < c) {
			result = c;
		}
		else{
			result = a;
		}
	}
	else{
		result = a;
	}

	std::cout << result << " eh o maior" << '\n';

	return 0;
}

File: URI Online Judg/Beginner/URI-Python/1013.py
import math

value = input().split()

a, b, c = value
a, b, c = int(a), int(b), int(c)

maior = (a + b + abs(a - b)) / 2
result = (maior + c + abs(maior - c)) / 2

print("%d eh o maior"%result)
